caveclient/skeletonservice.py

```diff
@@ -107,17 +107,29 @@ def __init__(
             over_client=over_client,
         )
     
+    def get_skeleton_info(
```

Comment:
needs a doc string

Reply:
What actually in the skeleton info endpoint? It doesn't take a root id, so I take it it's not info for a particular skeleton. Consider renaming to be more descriptive.

Reply:
This followed other patterns. There was always a top-level info REST endpoint and I just mapped it onto a client endpoint. See SkeletonService:apy.py:204 for the origin story. I'm not sure of the best way to structure these various accesses to the system.

Reply:
Got it. This seems like an info file for neuroglancer specifically — I'm not totally sure it makes sense an as endpoint in the client, though. Would be curious about the thoughts from others on this one.

Reply:
Not sure why this conversation got marked resolved. This info JSON is a specific preamble necessary for the Neuroglancer Precomputed Skeleton format, but is not particularly readable or generic basic information. I don't think it needs to be a client function.

Reply:
I closed it because I added a doc string which was the original issue. If it should be removed entirely, ok.

Reply:
I removed it.